### PR TITLE
Devex 206 wip

### PR DESCRIPTION
**Pull request by mrinsin**

### CODE REVIEW

File: web-client/src/presenter/actions/setDocumentSelectedForScanAction.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { state } from 'cerebral';
+
+/**
+ * returns a callback function that sets documentType on state
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.store the cerebral store object used for setting documentSelectedForScan
```
Comment (matthopson) [inline web-client/src/presenter/actions/setDocumentSelectedForScanAction.js]: nit: jsdocs / params

File: web-client/src/presenter/actions/setShowAllLocationsFalseAction.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+/**
+ * initiates the document to be served
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context used for getting the getUser use case
+ * @param {Function} providers.get the cerebral get function
+ * @param {object} providers.router the riot.router object containing the createObjectURL function
+ * @returns {object} the user
+ */
```
Comment (matthopson) [inline web-client/src/presenter/actions/setShowAllLocationsFalseAction.js:9]: nit: jsdocs

File: web-client/src/presenter/actions/updateSearchTermAction.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+/**
+ * initiates the document to be served
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context used for getting the getUser use case
+ * @param {Function} providers.get the cerebral get function
+ * @param {object} providers.router the riot.router object containing the createObjectURL function
+ * @returns {object} the user
+ */
```
Comment (matthopson) [inline web-client/src/presenter/actions/updateSearchTermAction.js:10]: jsdocs

File: web-client/src/presenter/actions/updateSearchTermAction.test.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { runAction } from 'cerebral/test';
+import { updateSearchTermAction } from './updateSearchTermAction';
+
+describe('updateSearchTermAction', () => {
+  const mockSearchTerm = 'order search';
+
+  it('state.header.searchTerm to the value of props.searchTerm', async () => {
```
Comment (matthopson) [inline web-client/src/presenter/actions/updateSearchTermAction.test.js]: little nit in test desc: "sets"

File: web-client/src/presenter/sequences/setDocumentUploadModeSequence.js [hunk 5]
```diff
@@ -1,3 +1,3 @@
 import { setDocumentUploadModeAction } from '../actions/setDocumentUploadModeAction';
 
-export const setDocumentUploadModeSequence = [setDocumentUploadModeAction];
+export const setDocumentUploadModeSequence = [setDocumentUploadModeAction()];
```
Comment (matthopson) [inline web-client/src/presenter/sequences/setDocumentUploadModeSequence.js:3]: I'm not following this change. Is this a factory?

Comment (rachelschneiderman) [reply web-client/src/presenter/sequences/setDocumentUploadModeSequence.js:3]: It is! The factory sets the value of `state.currentViewMetadata.documentUploadMode` from `props.documentUploadMode` or the argument passed in when `props.documentUploadMode` is undefined

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**